src/plugins/data_source_management/public/components/direct_query_data_sources_components/acceleration_creation/create/create_acceleration.test.tsx

```diff
@@ -26,6 +26,7 @@ const mockNotifications = {
   toasts: {
     addWarning: jest.fn(),
     addSuccess: jest.fn(),
+    addDanger: jest.fn(),
```

Comment:
Nit: there are common mocks from core, we can leverage that.

Reply:
Good suggestion. For the latest test cases I just added, I switched to use `coreMock` instead. And for this one and the rest of the entire test suites of this plugin, lets open a separate PR as follow up.